src/channels/__init__.py

```diff
@@ -240,24 +240,31 @@ def setChannelPan(index: int, pan: float, pickupMode: int = midi.PIM_None) -> No
 
 
 def getChannelPitch(index: int, mode: int = 0) -> 'float | int':
-    """Returns the pitch of the channel at `index`. The `mode` parameter is used
-    to determine the type of pitch returned. 
-    
-    HELP WANTED: What do the `mode` parameter options mean?
-    
+    """Returns the current pitch bend (or range) of the channel at `index`.
+    The `mode` parameter is used to determine the type of pitch returned.
+
     ## Args:
     * `index` (`int`): index of channel
-     
+
     * `mode` (`int`, optional):
-          * `1`: return value in semitones
 
-          * `2`: return value pitch range???
+        * `0` (default): return the current pitch bend as a factor of the current
+            range (usually `-1.0` to `1.0`). Larger values might be reached if the
+            pitch is automated with events, for example.
+
+        * `1`: return the current pitch offset in cents.
+
+            BUG: Official API docs incorrectly state "semitones".
+
+        * `2`: return the current pitch range in semitones.
+
+            BUG: This is not guaranteed to be correct. See `getChannelPitch`.
```

Comment:
This seems to be self-referencing?

Reply:
Nice catch, `setChannelPitch` is the actual intended reference.